ui/src/app/pages/admin/workspace/about-publish-consent-modal.tsx

```diff
@@ -31,18 +35,48 @@ export const AboutPublishConsentModal = (props: Props) => {
       [checkboxId]: checked,
     }));
   };
+
+  const [publishing, setPublishing] = useState(false);
```

Comment:
should we rename everywhere we see "publish" here?  (the filename and component name too)

Reply:
I am not sure about the user facing term before talking to Joy and Moira so i started out with API.Since the tab which shows all 'PUBLISHED' workspace is **Featured** Workspace i prefer using the term Featured.